fast_node.go

```diff
@@ -53,6 +54,12 @@ func (fn *FastNode) GetKey() []byte {
 	return fn.key
 }
 
+func (fn *FastNode) GetFullSize() int {
```

Comment:
Can we add a godoc describing the value returned here? It's not super clear to me.